syntax = "proto2";

package proto.db;

message call_command {
	required uint32 channel_id = 1;
	required string sql = 2;
	repeated bytes args = 3;
}